lib/livebook/teams.ex

```diff
@@ -153,6 +153,60 @@ defmodule Livebook.Teams do
     end
   end
 
+  @doc """
+  Creates a File System.
+
+  With success, returns the response from Livebook Teams API.
+  Otherwise, it will return an error tuple with changeset.
```

Comment:
On success we return `:ok`, so it's not really accurate, but either way the typespec should be enough :)

```suggestion
```